CMakeLists.txt

```diff
@@ -237,14 +239,68 @@ set(CPACK_PACKAGE_VERSION_PATCH ${VERSION_PATCH})
 set(CPACK_PACKAGE_VENDOR "celeron55")
 set(CPACK_PACKAGE_CONTACT "Perttu Ahola <celeron55@gmail.com>")
 
+include(CPackComponent)
+
+cpack_add_component(Docs
+	DISPLAY_NAME "Documentation"
+	DESCRIPTION "Documentation about Minetest and Minetest modding"
+)
+
+cpack_add_component(SUBGAME_MINETEST_GAME
+	DISPLAY_NAME "Minetest Game"
+	DESCRIPTION "The official subgame for the Minetest engine, that can easily extended by mods."
+	GROUP "Subgames"
+)
+
+cpack_add_component(SUBGAME_MINIMAL
+	DISPLAY_NAME "Minimal development test"
+	DESCRIPTION "A minimal subgame helping to develop the engine."
+	DISABLED #DISABLED does not mean it is disabled, it means its not selected on default!
```

Comment:
In my opinion that comment is really important, because if something is disabled (e.g. a button, or in this case a checkbox) it means it is grayed out and not accessible/changeable/clickable by the user. But for cmake it means just unchecked